internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+type PullRequestEvent struct {
+	Action      string      `json:"action"`
+	PullRequest PullRequest `json:"pull_request"`
+	Repository  Repository  `json:"repository"`
+}
+
+type GitHubUser struct {
+	Login string `json:"login"`
+	Id    int    `json:"id"`
+	Url   string `json:"url"`
+}
+
```

Comment:
The `Body` field in `PullRequest` may require sanitization or validation to prevent security vulnerabilities such as XSS if rendered as HTML in any UI. Ensure appropriate handling for user-generated content.